utils.sh

```diff
@@ -57,6 +57,10 @@ function create_cluster() {
 
     $OPENSHIFT_INSTALLER --dir "${assets_dir}" --log-level=debug create manifests
 
+    # For CI and nightly releases we need to remove the channel property to
+    # prevent critical alerts.
+    sed -i '/^  channel:/d' "${assets_dir}/manifests/cvo-overrides.yaml"
```

Comment:
Should this be conditional, since we do support a release type of "ga" in addition to the "ci" and "nightly" options?

https://github.com/openshift-metal3/dev-scripts/blob/master/common.sh#L117